server/schema/published_schedule.sql

```diff
@@ -0,0 +1,16 @@
+DROP TABLE [IF EXISTS] published_schedule;
+
+CREATE TYPE year AS ENUM ('Junior','Senior');
+
+CREATE TABLE [IF NOT EXISTS] published_schedule (
```

Comment:
the brackets aren't needed here. the brackets in the tutorial indicate optional fields but when you actually use those options, you don't use the brackets